website/templates/profile/account.mako

```diff
@@ -65,8 +65,12 @@
                                 <tr>
                                     <td colspan="3">
                                         <form data-bind="submit: addEmail">
-                                            <p>To merge an existing account with this one or to log in with multiple email addresses, add an alternate email address below.</p>
-
+                                            <p>
+                                            To merge an existing account with this one or to log in with multiple email addresses, add an alternate email address below.
+                                            <a id="addDashboardFolder" class="fa fa-info-circle social-icons" data-bind="tooltip: {title: 'Merging accounts will move all projects and components associated with two emails into one account. All projects and components will be displayed under the email address listed as primary.',
```

Comment:
I think the `social-icons` class might have been removed -- it doesn't seem to add any additional formatting and I can't seem to find where it's declared. Can you please confirm?